Makefile

```diff
@@ -1,5 +1,5 @@
 swag:
-	swag init --parseDependency --parseDepth=5 -d ./cmd/rest -o ./cmd/rest/docs
+	swag init --parseDependency --parseDepth=5
```

Comment:
Does it work without `--parseInternals`?

Reply:
Yes